docker-compose.yaml

```diff
@@ -374,6 +374,12 @@ services:
       - ./scripts/docker-compose/imports/smartdocuments-wiremock/mappings:/home/wiremock/mappings
     entrypoint: [ "/docker-entrypoint.sh", "--global-response-templating", "--disable-gzip", "--verbose" ]
 
+  office-converter:
+    image: ghcr.io/eugenmayer/kontextwork-converter:1.5.0
+    ports:
+      - "9999:8080"
```

Comment:
Maakt niet zo heel veel uit maar ik zou liever in de 8xxx range blijven voor alle '8080' gemapte poorten zeg maar zoals we ook doen voor de andere Docker containers. 8083 of zo iets?